react-frontend/src/components/Nav/breadcrumbs.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
a component like this I'd love to see tested really really well. not comfortable releasing this without functional tests.